### PR TITLE
fix optional chaining op

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte [hunk 1]
```diff
@@ -41,7 +41,7 @@
   {/if}
 
   {#if level !== "live"}
-    <div data-testid="level" class="service-level service-level--{level?.toLowerCase()}">
+    <div data-testid="level" class="service-level service-level--{level.toLowerCase()}">
```
Comment (bluekurinji) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:44]: The change is fine. You could also use css for this.   text-transform: capitalize;

**Review by bluekurinji — APPROVED** [review]

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte (hunk 1 above)
Comment (chrisolsen) [reply libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:44]: For html text content you can, but not for css class names.

**Review by ravi-dio — APPROVED** [review]

**Review by ravi-dio — APPROVED** [review]

**Pull request merged**